src/histolab/filters/morphological_filters_functional.py

```diff
@@ -23,7 +23,7 @@
 
 
 def remove_small_objects(
-    np_img: np.ndarray,
+    mask_img: np.ndarray,
```

Comment:
to be compliant with the other names like `np_mask` i would use the same name here